example/ck_tile/20_grouped_convolution/grouped_convolution_forward_large_tensor_invoker.hpp

```diff
@@ -363,40 +364,43 @@ struct GroupedConvolutionForwardInvoker
                           << ", Vector size C: " << ConvEpilogue::GetVectorSizeC() << std::endl;
             }
 
-            ave_time = ck_tile::launch_kernel(
```

Comment:
it also doesn't seem necessary to share a variable between all the callbacks being passed around to just store the result